### DISCUSSION TITLE
DUDA coordenada X e Y parte 1

### DISCUSSION THREAD

**Discussion by Lexiesm** [IIC2133-PUC/2024-2#147] [Tarea 2] [answered]
Hola! espero estén bien!! me entró una duda en la parte de las coordenadas de los pinguinos, cual sería el 0 de los ejes? sería algo así?: 
![imagen](https://github.com/user-attachments/assets/0aca4ee1-18a3-436c-b66f-6e2b9a851dfb)

(perdón lo feo del dibujo) onda el nodo 1 estaría en la posición (3,3)? cómo se esta midiendo el punto (0,0)? o lo "defino" yo??
Gracias de antemano!

**Comment by sebacarrascop** [comment, marked as answer]
holaa!, no entendi muy bien tu pregunta, pero a los pingüinos al crearlos se les da un eje X y un eje Y, y se hacen consultas respecto a su posición. El como tú manejes los árboles para representar coordenadas es decisión propia de diseño. Si algo no queda claro sigue preguntando!
